### PR TITLE
Add Skull/unskull notifications

### PR DESCRIPTION
**Pull request by Trevor159**
Useful at revs, idk where else.

### CODE REVIEW

**Comment by 15987632** [comment]
I have a skull timer PR also, I don't have the knowledge to make it 100% accurate when reseting your skull in pvp which is why that one is on hold for now. Yes I can move this to idle notifications but personally I didn't feel this was fitting.

**Comment by 15987632** [comment]
I'm pretty sure i'll keep the skull timer as a separate PR, this is in a finished state and the skull timer is not. And the skull timer would do into a completely separate plugin anyways.

**Comment by Adam-** [comment]
I guess it is convenient to put this in the idle notifier but I'm not sure how many people skull when idle.

File: runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java [hunk 1]
```diff
@@ -64,11 +64,33 @@ default boolean logoutIdle()
 		return true;
 	}
 
+	@ConfigItem(
+			position = 4,
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java]: Fix indenting

File: runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java [hunk 2]
```diff
@@ -328,6 +333,29 @@ public void onHitsplatApplied(HitsplatApplied event)
 	public void onGameTick(GameTick event)
 	{
 		final Player local = client.getLocalPlayer();
+		SkullIcon currentTickSkull = local.getSkullIcon();
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java]: Probably all of this logic should be in its own method.

**Comment by Trevor159** [comment]
I think I was the only person who ever wanted this, probably useless.

**Pull request closed without merging**